BuildResidentialHPXML/measure.rb

```diff
@@ -1382,6 +1420,74 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setUnits('W')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument.makeChoiceArgument('heat_pump_perf_data_heating_cap_type', perf_data_capacity_type_choices, false)
```

Comment:
I tend to think we should just have `heating_perf_data_foo` and `cooling_perf_data_foo` inputs, which means we don't need duplicate sets of cooling inputs. In general, the measure doesn't support having both an AC and a HP, though there might be some edge cases where it's possible...?